### PR TITLE
Implement Find in page and Add to home screen updates

### PR DESCRIPTION
**Pull request by ionutbedregeanu**
### For #4937 

Update Find in page component position and colors according to theme mode
Update Add to home screen dialog design
[Fix] Page title is not displayed in Add home screen dialog because current tab object is updated after dialog show event

#### For context menus changes I created [a new ticket](https://github.com/mozilla-mobile/focus-android/issues/5098)

### CODE REVIEW

File: app/src/main/res/values/strings.xml
Comment (jonalmeida) [inline app/src/main/res/values/strings.xml:478]: Please copy over the comment from the 476 to above this string as well. Our automation extracts the strings file with the comment above each string to provide localizers context for translating.

Comment (jonalmeida) [reply app/src/main/res/values/strings.xml:478]: Same case for `dialog_addtohomescreen_action_no`.

File: app/src/main/res/values-night/colors.xml
Comment (jonalmeida) [inline app/src/main/res/values-night/colors.xml:10]: I imagined the intention of adding these colours is so that we switch over to using the themed colours for each mode?

Wouldn't we default to the normal `values/colors.xml` if we remove the `values-night/colors.xml`?

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt
Comment (ionutbedregeanu) [reply app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt:250]: Seems that some updates were made. @brampitoyo At this [Figma link](https://www.figma.com/file/2V6wgf9fRFdYfLaf5c3oPj/Product-UI?node-id=2133%3A282824) we can find Find in page bar at two differerent positions. Can you tell us which one is the correct one?

Screenshot 2021-08-12 at 10 08 01

Screenshot 2021-08-12 at 10 09 29

Comment (jonalmeida) [reply app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt:250]: In the interest of avoiding this PR from going stale, let's land this while we wait for final UX review. Updating this change shouldn't be difficult after it lands.

**Review by jonalmeida — APPROVED** [review]
✅

**Comment by pocmo** [comment]
> ✅

You didn't actually prove the PR though - That's what Mergify is waiting for here :)

**Review by mcarare — APPROVED** [review]

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt
Comment (brampitoyo) [reply app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt:254]: @ionutbedregeanu @jonalmeida We should use the top-aligned “Find in page” UI.

When the toolbar is moved to the bottom of the screen (maybe we will plan for this in the future), then the “Find in page” UI will also move to the bottom, so it will be located above the keyboard.

**Pull request merged**
